db.js

```diff
@@ -37,10 +37,32 @@ var client = couchdb.createClient(
     settings.username,
     settings.password
 );
-module.exports = client.db(settings.db);
+var db = client.db(settings.db);
+
+// Fix for 0.4 : https://github.com/medic/medic-projects/issues/1178#issuecomment-273550990
```

Comment:
How about some english here instead of a link? I think if you're going to comment, then comment otherwise I don't want to wade through complicated issues.  Give me your succinct wisdom please!